### ISSUE TITLE
Exception has occurred: FileNotFoundError

### ISSUE THREAD

**Issue by IvanIsak2000** [mb6ockatf/doodle-jump#28]
When I launch main.py , I get an error message :(

![image](https://user-images.githubusercontent.com/79650307/226668453-6532e512-4f25-453e-a816-6521588e2524.png)

**Comment by mb6ockatf** [comment]
@IvanIsak2000 please check if you're running `python3 src/main.py` in project's root folder, not `python3 main.py` in `/src` folder.
That's because sprites are located out of `/src` folder, in `/data`

**Comment by mb6ockatf** [comment]
@IvanIsak2000 btw are you Russian? Just noticed your Windows interface is Russian

**Comment by IvanIsak2000** [comment]
> @IvanIsak2000 please check if you're running `python3 src/main.py` in project's root folder, not `python3 main.py` in `/src` folder. That's because sprites are located out of `/src` folder, in `/data`


That's right, I open main and gives an error :(

**Comment by IvanIsak2000** [comment]
@mb6ockatf 

I don't understand, the ``main.py`` file is in ``src``, is that correct?

![image](https://user-images.githubusercontent.com/79650307/226686420-e6c62176-e265-4302-8095-702fb56d872b.png)

**Comment by mb6ockatf** [comment]
> @mb6ockatf
> 
> I don't understand, the `main.py` file is in `src`, is that correct?
> 
> ![image](https://user-images.githubusercontent.com/79650307/226686420-e6c62176-e265-4302-8095-702fb56d872b.png)

Yes, that's correct. And this file `main.py` is supposed to be launched from project's root folder
i.e.
```bash
$ pwd  # check current directory path
/home/username/projects/doodle-jump  # we're in project's root
$ python3 src/main.py  # launch the game
# starts successfully
```

Otherwise, relative imports don't work. Here's an error you get:
```bash
$ pwd  # check current directory path
/home/mb6ockatf/projects/doodle-jump/src  # note, we're now in `doodle-jump/src`
$ python3 main.py  # raises an error, as it cannot access resources in `doodle-jump/data`
```
```python
file 'data/bird_left.png' not found
Traceback (most recent call last):
  File "/home/username/projects/doodle-jump/src/main.py", line 108, in <module>
    gamekeeper = Gamekeeper()
  File "/home/username/projects/doodle-jump/src/main.py", line 35, in __init__
    self.hopalong = Hopalong(self.players, sprite_name, display_sizes)
  File "/home/username/projects/doodle-jump/src/sprites/hopalong.py", line 36, in __init__
    sprite_left = load_image(self.name + "_left.png")
  File "/home/username/projects/doodle-jump/src/tools/funcs.py", line 16, in load_image
    image = pygame.image.load(fullname)
FileNotFoundError: No file 'data/bird_left.png' found in working directory '/home/username/projects/doodle-jump/src'.
```

------

I tried my best when organising imports in this project. If you find a way to improve it, @IvanIsak2000, please create a PR for that

**Comment by mb6ockatf** [comment]
<details>
<summary><h1>Conclusion</h1></summary>
<h2>What can be done:</h2>

- specify the way the game is supposed to be launched more precisely in `README.md`
- improve imports

- create a bash script in project's root, and write in README.md that the game is supposed to be launched this way

One of these should fix it

</details>

**Comment by IvanIsak2000** [comment]
> > @mb6ockatf
> > I don't understand, the `main.py` file is in `src`, is that correct?
> > ![image](https://user-images.githubusercontent.com/79650307/226686420-e6c62176-e265-4302-8095-702fb56d872b.png)
> 
> Yes, that's correct. And this file `main.py` is supposed to be launched from project's root folder i.e.
> 
> ```shell
> $ pwd  # check current directory path
> /home/username/projects/doodle-jump  # we're in project's root
> $ python3 src/main.py  # launch the game
> # starts successfully
> ```
> 
> Otherwise, relative imports don't work. Here's an error you get:
> 
> ```shell
> $ pwd  # check current directory path
> /home/mb6ockatf/projects/doodle-jump/src  # note, we're now in `doodle-jump/src`
> $ python3 main.py  # raises an error, as it cannot access resources in `doodle-jump/data`
> ```
> 
> ```python
> file 'data/bird_left.png' not found
> Traceback (most recent call last):
>   File "/home/username/projects/doodle-jump/src/main.py", line 108, in <module>
>     gamekeeper = Gamekeeper()
>   File "/home/username/projects/doodle-jump/src/main.py", line 35, in __init__
>     self.hopalong = Hopalong(self.players, sprite_name, display_sizes)
>   File "/home/username/projects/doodle-jump/src/sprites/hopalong.py", line 36, in __init__
>     sprite_left = load_image(self.name + "_left.png")
>   File "/home/username/projects/doodle-jump/src/tools/funcs.py", line 16, in load_image
>     image = pygame.image.load(fullname)
> FileNotFoundError: No file 'data/bird_left.png' found in working directory '/home/username/projects/doodle-jump/src'.
> ```
> 
> I tried my best when organising imports in this project. If you find a way to improve it, @IvanIsak2000, please create a PR for that

OK, I launched the app, as you said, it's great!

**Comment by mb6ockatf** [comment]
actialy fixed in #30
